Portfolio/SystemDesign.md

```diff
@@ -3,15 +3,19 @@
 # 1. System Design [40 pts] :
 ## a. Architecture of the entire system
 
-Architecture of the system uses a central controller database API that handles receiving and sending requests, using MQTT protocol, to communicate between devices. Three key devices of system:
+Our Do or Die fitness tracker is an Internet of Things (IoT) product that we designed to operate across three different platforms; the M5Stack, the Web, and a Management Dashboard. The system architecture uses a central controller API which communicates with the database, and handles receiving and sending requests using MQTT protocol to communicate between devices. Three key devices of system:
```

Comment:
just change the three key devices / separate the other sections maybe? hard to see in the GitHub review panel

Reply:
as in rather than three key devices / or do like our system is structured : (hard to see if it is split into three and then other aspects here but you know) love you @a-shmyg xoxox